guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -15,9 +15,25 @@ However, {Project} does not remove the back-end services on the operating system
 --foreman-proxy-tftp false
 ----
 
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
+. Stop and mask the TFTP and DNS services (FIXME Please confirm that DNS needs to be stopped and masked too):
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# systemctl mask --now tftp.service tftp.socket systemd-resolved.service
+----
+
+. Optional: On RHEL7 hosts, disable the eXtended Internet Services Daemon (xinetd) and delete the `xinetd` package:
```

Comment:
This procedure is about the Foreman/Katello platform which does not run on EL7/8 anymore. If you absolutely _need_ these lines, then please create a follow-up PR that only targets the relevant X.Y branches.